src/containers/Signin/signinActions.js

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+
+//actions creators
+export function fetchSigninToken(username, password) {
+    return  {
+        type: "FETCH_USERINFO",
+        payload: axios({ 
+            method: "post",
+            url: "https://frontend-challenge-2.herokuapp.com/rest-auth/login/",
+            data: {
+                username: username,
+                password: password
+            }
+        })
+        .then(function (response) {
```

Comment:
As I told you in person, try to avoid using these `then` and `catch`, handle these in reducer itself.